### PR TITLE
makes sleep assessment section look better on desktop 

### PR DESCRIPTION
**Pull request by Danwhy**
ref #139
old: 
screen shot 2017-05-11 at 11 11 21
new: 
screen shot 2017-05-11 at 11 53 10

### CODE REVIEW

File: web/templates/homepage/index.html.eex [hunk 1]
```diff
@@ -5,8 +5,8 @@
   </div>
 </section>
 
-<section class="pt1 pb3 pt5-l ph4 ph5-m ph6-l lm-bg-dark-blue tc">
-  <p class="white tl">
+<section class="pt1 pb3 ph4 ph5-m ph6-l lm-bg-dark-blue tc">
+  <p class="white tl tc-l">
```
Comment (SimonLab) [inline web/templates/homepage/index.html.eex]: Just to confirm, tc-l will only centre the text on large screeen, does the text need to be centre on mobile too?

**Review by SimonLab — COMMENTED** [review]
Looks good on desktop, just a small question/comment for mobile version @Danwhy

File: web/templates/homepage/index.html.eex (hunk 1 above)
Comment (Danwhy) [inline web/templates/homepage/index.html.eex]: No, text is left on mobile: https://github.com/LDMW/app/issues/77#issuecomment-299870609

**Review by SimonLab — APPROVED** [review]
:+1:

**Pull request merged**